packages/react-components/virtualizer/stories/Virtualizer/Default.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Virtualizer } from '@fluentui/react-components/unstable';
+import { makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    overflowAnchor: 'none',
+    overflowY: 'auto',
+    width: '100%',
+    height: '100%',
+    maxHeight: '750px',
```

Comment:
It would be nice to export these styles as an object for easier use

https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-theme/src/global/typographyStyles.ts

Reply:
Are we ok to export these via Virtualizer library, or should we keep them in react-theme?